src/test/java/com/cdancy/bitbucket/rest/features/BranchApiLiveTest.java

```diff
@@ -17,17 +17,16 @@
 
```

Comment:
Another LiveTest to check for errors? Maybe named something like `testGetBranchModelConfigurationOnError()`?